lib/plugins/physics.js

```diff
@@ -128,7 +134,7 @@ function inject(bot) {
 
     // limit speed
     var currentMaxGroundSpeed;
-    var underBlock=bot.blockAt(pos.offset(0,-1,0));
+    var underBlock=bot.blockAt(new Vec3(pos.x,Math.ceil(pos.y-1.002),pos.z));
```

Comment:
Haha I know :D 
That's because of https://github.com/andrewrk/mineflayer/blob/e3573541a5caf8286e07c50a1c61fa0f5d94b9c0/lib/plugins/physics.js#L155
So the bot position is a little bit too high, removing 0.002 gives the correct ceiling.

Not sure if there's a better way here, probably.